src/main/java/com/faforever/api/user/NameRecordRepository.java

```diff
@@ -12,6 +12,6 @@
   @Query(value = "SELECT datediff(now(), change_time) FROM name_history WHERE user_id = :userId AND datediff(now(), change_time) <= :maximumDaysAgo ORDER BY change_time DESC LIMIT 1", nativeQuery = true)
   Optional<BigInteger> getDaysSinceLastNewRecord(@Param("userId") Integer userId, @Param("maximumDaysAgo") Integer maximumDaysAgo);
 
-  @Query(value = "SELECT user_id FROM name_history WHERE previous_name = :name AND now() > date_add(change_time, INTERVAL :months MONTH) ORDER BY change_time DESC LIMIT 1", nativeQuery = true)
+  @Query(value = "SELECT user_id FROM name_history WHERE previous_name = :name AND now() < date_add(change_time, INTERVAL :months MONTH) ORDER BY change_time DESC LIMIT 1", nativeQuery = true)
```

Comment:
as I said in the issue I think it would be better to do `now() - x month < change_date` if we use or will be using index on this column

Reply:
Yes, good suggestion.